server/db-patches/patch-2-3.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE data ADD url TEXT;
```

Comment:
Maybe this should be NOT NULL?  Of course the UPDATE below has to run first, but it could then have `ALTER TABLE data ALTER COLUMN url SET NOT NULL` (I think)